tools/lsp/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "slint-lsp"
 description = "A language server protocol implementation for Slint"
 authors = ["Slint Developers <info@slint.dev>"]
 edition.workspace = true
-homepage = "https://slint.dev"
+homepage.workspace = true
```

Comment:
This doesn't work because some of our tooling don't support this syntax  (see commit history)